enhancements/extending-ldap.md

```diff
@@ -85,3 +85,31 @@ defined LDAP entity.
 ## Implementation History
 
 - 2022-07-08 Initial proposal
+
+
+### Implementation
+
+#### Superusers
+
+In addition to current superuser capabilities, this implementation will add the ability for superusers to access contents from outside its permission scope. This is a very specific use case and would only be enabled if the given FEATURE flag is set. This would allow superusers, on top of taking ownership of other namespaces, read, write, and delete contents from other namespaces, without breaking backward compatibility (i.e exising installations should be able to opt-in to get this behavior).
+
+In terms of permission, access to resources not owned by the user would be granted based on whether or not that user has the existing superuser grant (scope) or not. This means that superusers would essentially bypass standard authentication in these cases. TODO: security implications.
```

Comment:
Does this include user orgs?

Reply:
Are there any concerns about a superuser being able to sidestep auditing?

Reply:
Is it really sidestepping auditing? I read here that it would be sidestepping standard authentication. And I am not sure why it would have anything to do how the user authenticates. The superuser still always has to authenticate with credentials. Is it possible that this was meant to say "authorization"? If so, the standard authorization should be adjusted to always grant the superuser to anything it request. It is the _superuser_ after all.

Reply:
> Is it possible that this was meant to say "authorization"?

I think so. Superuser actions should still be audited though, but giving them the ability to do _anything_ makes it sound like they could then tinker with auditing...which would defeat the purpose of auditing the actions.